src/GraphQL.SystemTextJson/ApolloTraceJsonConverter.cs

```diff
@@ -22,8 +22,15 @@ public class ApolloTraceJsonConverter : JsonConverter<ApolloTrace>
             PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
         };
 
+        public ApolloTraceJsonConverter()
+        {
+            var converter = new ROMJsonConverter();
+            _optionsNoIndent.Converters.Add(converter);
+            _optionsIndent.Converters.Add(converter);
+        }
+
         /// <inheritdoc/>
-        public override void Write(Utf8JsonWriter writer, ApolloTrace value, JsonSerializerOptions options)
+        public override void Write(Utf8JsonWriter writer, ApolloTrace value, JsonSerializerOptions options) //TODO: options ignored
```

Comment:
It is right?

Reply:
This is by design, to enforce camelCase.

Reply:
OK, I'll add comment.

Reply:
done